day6/cpp/anagrams.cpp

```diff
@@ -0,0 +1,38 @@
+/**
+ * @author Bhanu0202
+ * @date 27/12/2018
+ */
+
+#include <bits/stdc++.h>
+using namespace std;
+
+int main() {
+	string sa, sb;
+	cin >> sa >>sb;
+	int la = sa.length();
+	int lb = sb.length();
+	int count[26] = {0};
+	if(la != lb){
+	    cout << "Invalid Input";
+	}
+	else{
+	    int flag = 0, i;
+	    transform(sa.begin(), sa.end(), sa.begin(), ::tolower);
```

Comment:
Dont convert strings to lower case.
README file says **Do not ignore the letter case**, so _**A**bc_ and _**a**bc_ are not anagrams.

Reply:
Thanks for the correction. I will update it soon.